docs/getting-started/react-native-ios.md

```diff
@@ -151,12 +185,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     #if FB_SONARKIT_ENABLED
       let client = FlipperClient.shared()
       let layoutDescriptorMapper = SKDescriptorMapper(defaults: ())
-      FlipperKitLayoutComponentKitSupport.setUpWith(layoutDescriptorMapper)
       client?.add(FlipperKitLayoutPlugin(rootNode: application, with: layoutDescriptorMapper!))
       client?.add(FKUserDefaultsPlugin(suiteName: nil))
       client?.add(FlipperKitReactPlugin())
       client?.add(FlipperKitNetworkPlugin(networkAdapter: SKIOSNetworkAdapter()))
-      client?.add(FlipperReactPerformancePlugin.sharedInstance())
```

Comment:
Same for here, you will have to add `pod FlipperReactPerformancePlugin` to make it work.